lib/main_common.pm

```diff
@@ -1758,6 +1758,7 @@ sub load_x11_webbrowser_extra {
     loadtest "x11/firefox/firefox_ssl";
     loadtest "x11/firefox/firefox_emaillink";
     loadtest "x11/firefox/firefox_plugins";
+    loadtest "console/ca_certificates_mozilla";
```

Comment:
Verification run shows test inclusion in some cloned wireshark extratests,
but here it is included between firefox tests.

Reply:
Positioning of the test in indeed different as on http://onalmpantis.suse.de/tests/7 I put it first in order to not have to run the whole testsuite to evaluate if it works.

Reply:
It is console test, it should be in console tests as you have in verification run and not in mixed in x11 test in firefox.  It should be placed somewhere after consoletest_setup/hostname/zypper_lr_validate.  Do verification runs for SLE12-SP3 too.
Would be great to verify funcionality on openSUSE TW run, because the code is shared with openSUSE too (https://openqa.opensuse.org/tests/759347).

Reply:
No, for two reasons:
1. Switching to console tests in the middle of graphical tests is not good idea, it should be at least somehow separated and consoletest_setup  should be done.

2. ca-certificates-mozilla is SLES package, load_x11_webbrowser_extra is loaded only for firefox/browser regression testing in SLED

Reply:
Well, then how about to put it into a specific test scenario? But please not the generic "consoletests"